### PR TITLE
support for python phases in zingg.sh

### PR DESCRIPTION
**Pull request by navinrathore**
Tested with cluster as well.
If phase is not on the list of dictionary (hard coded), standard Zingg execution take place. In case of python phase, it shall be executed. only phase option is used as of now.

### CODE REVIEW

File: scripts/zingg.sh [hunk 1]
```diff
@@ -3,34 +3,77 @@
...
+	esac
+done
+set -- "${POSITIONAL_ARGS[@]}" # restore positional parameters
+
 OPTION_SPARK_CONF+=$(read_zingg_conf)
+
+# if it is a python phase
+if [[ $RUN_PYTHON_PHASE -eq 0 ]]; then
+	echo "Executing python phase: ${PHASE} (${PYTHON_PHASE_EXE})"
+	$SPARK_HOME/bin/spark-submit --master $SPARK_MASTER --jars=$ZINGG_JARS --conf spark.driver.extraClassPath=$ZINGG_JARS --conf spark.executor.extraClassPath=$ZINGG_JARS $PYTHON_PHASE_EXE --email $EMAIL --license $LICENSE
```
Comment (sonalgoyal) [inline scripts/zingg.sh:73]: we need to pass the remaining arguments to the python phase as well.

Comment (sonalgoyal) [reply scripts/zingg.sh:73]: how about passing the other options? (ALL_OPTIONS)

Comment (sonalgoyal) [reply scripts/zingg.sh:73]: and --py-files?

**Review by sonalgoyal — CHANGES_REQUESTED** [review]

**Comment by navinrathore** [comment]
#260

**Pull request merged**